src/main/java/world/bentobox/bentobox/hooks/LangUtilsHook.java

```diff
@@ -11,79 +12,38 @@
 import org.bukkit.entity.TropicalFish;
 import org.bukkit.entity.Villager;
 import org.bukkit.inventory.ItemStack;
-import org.bukkit.inventory.meta.ItemMeta;
 import org.bukkit.inventory.meta.TropicalFishBucketMeta;
-import org.bukkit.plugin.Plugin;
 import org.bukkit.potion.PotionEffect;
 import org.bukkit.potion.PotionEffectType;
 import org.bukkit.potion.PotionType;
 import org.jetbrains.annotations.Nullable;
-import world.bentobox.bentobox.BentoBox;
+
+import com.meowj.langutils.lang.LanguageHelper;
+
 import world.bentobox.bentobox.api.hooks.Hook;
 import world.bentobox.bentobox.api.user.User;
-import world.bentobox.bentobox.util.Util;
-
-import java.util.Locale;
-import java.util.Map.Entry;
-import java.util.logging.Logger;
 
 /**
  * @author ApacheZy
- * @since 1.6.0
+ * @since 1.17.0
  */
-@SuppressWarnings("unused")
 public class LangUtilsHook extends Hook {
 
-    private static boolean hooked;
-
     public LangUtilsHook() {
         super("LangUtils", Material.BOOK);
     }
 
-    private static boolean doHook(Plugin plugin) {
-
-        // Because there are other plugins with the same name,
-        // we should check here whether it is the plugin we need.
-
-        if (plugin != null && plugin.isEnabled()) {
-
-            String tag = plugin.getConfig().getString("Extra-TAG");
-            if ("tag_r72EhIAL".equals(tag)) {
-                hooked = true;
-                return true;
-            }
-
-            Logger logger = BentoBox.getInstance().getLogger();
-            logger.warning("This LangUtils version is not available for BentoBox.");
-            logger.warning("Please go here to download the latest version:");
-            logger.warning("https://github.com/apachezy/LangUtils/releases");
-        }
-
-        hooked = false;
-        return false;
-    }
-
     @Override
     public boolean hook() {
-        return LangUtilsHook.doHook(getPlugin());
+        return getPlugin() != null && getPlugin().isEnabled() && getPlugin().getConfig().getString("Extra-TAG", "").equals("tag_r72EhIAL");
```

Comment:
Should be changed to:
`return getPlugin() != null && getPlugin().isEnabled() && "tag_r72EhIAL".equals(getPlugin().getConfig().getString("Extra-TAG", ""));`
Otherwise, a NullException may occur.